kubernetes/log-git-commit.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+# Logs the git commit (assumed to be saved in the GIT_COMMIT environment
+# variable) used for an experiment.
+
+OUTPUT_DIR=$1
+
+mkdir -p "$OUTPUT_DIR"
+COMMIT_FILE="$OUTPUT_DIR"/commit
+GIT_COMMIT=${GIT_COMMIT:-no_commit_found}
+
+if [ -f "$COMMIT_FILE" ]; then
+  # COMMIT_FILE already exists, but we may still want to log the commit if the
+  # commit has changed due to restarting a job.
+  LATEST_COMMIT=$(tail -n 1 "$COMMIT_FILE" | awk '{print $2}')
+  if [ "$LATEST_COMMIT" != "$GIT_COMMIT" ]; then
```

Comment:
You could simplify the code slightly by just doing an early-exit with `exit 0` here if `"$LATEST_COMMIT" == "$GIT_COMMIT"`, and otherwise falling through to appending to `$COMMIT_FILE` (lines 16 and 18 are duplicated)